torch/autograd/gradcheck.py

```diff
@@ -195,7 +196,22 @@ def _differentiable_outputs(x):
     return tuple(o for o in _as_tuple(x) if o.requires_grad)
 
 
-def gradcheck(func, inputs, eps=1e-6, atol=1e-5, rtol=1e-3, raise_exception=True, check_sparse_nnz=False, nondet_tol=0.0):
+# 'func' accepts a vararg of tensors, which isn't expressable in the type system at the moment.
+# If https://mypy.readthedocs.io/en/latest/additional_features.html?highlight=callable#extended-callable-types is accepted,
+# the '...' first argument of Callable can be replaced with VarArg(Tensor).
+# For now, we permit any input.
+# the '...' first argument of Callable can be replaced with VarArg(Tensor).
+# For now, we permit any input.
+def gradcheck(
+    func: Callable[..., Union[torch.Tensor, Tuple[torch.Tensor, ...]]],
```

Comment:
Takes a list of tuples as well (see `_as_tuple` call in function body); did you type this as `Tuple` instead of `Sequence` on purpose? If not, could use the same `_TensorOrTensors` you defined in `__init__.py`.

Reply:
or maybe that's related to the _vararg of tensors, which isn't expressable_ comment?

Reply:
I think the Vararg of tensors is referring to the `...` in Callable